web/src/user_status_ui.ts

```diff
@@ -105,12 +106,21 @@ export function clear_message(): void {
     const $field = input_field();
     $field.val("");
     $("#clear_status_message_button").prop("disabled", true);
+    hide_clear_status_button();
 }
 
 export function user_status_picker_open(): boolean {
     return $("#set-user-status-modal").length > 0;
 }
 
+export function hide_clear_status_button(): void {
```

Comment:
```suggestion
export function toggle_clear_status_button(): void {
```
Given that isn't only hiding the `x`.